.changeset/brave-cows-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-extension-dsl-data-space': patch
+---
+
+Allow viewing more informations about a data space when creating a new query in `Legend Query`, such as entitlement, project links (the project that the data space resides as well as the project that the data space refers to), test data, etc.
```

Comment:
What entitlement work did you do here. Don't see any within the dsl data space.

Reply:
The tab using `KeyIcon`, it's a placeholder, so meant to be WIP